public/integration/gallery-notebook.js

```diff
@@ -1,23 +1,23 @@
 $.ajaxSetup({ xhrFields: { withCredentials: true } });
 
 require(['base/js/utils','services/config'], function(utils,configmod) {
-    
+
```

Comment:
This entire file is just whitespace changes, don't touch this file in the merge, and in fact I don't think this is even used anymore